sdk/storage/Azure.Storage.Files.DataLake/tests/FileClientTests.cs

```diff
@@ -1009,7 +1009,7 @@ public async Task GetAccessControlAsync_Error()
             // Act
             await TestHelper.AssertExpectedExceptionAsync<RequestFailedException>(
                 file.GetAccessControlAsync(),
-                e => Assert.AreEqual("404", e.ErrorCode));
+                e => Assert.AreEqual("PathNotFound", e.ErrorCode));
```

Comment:
So for some Data Lake errors, we were previously returning numerical error codes.  This is inconsistent with the rest of the .NET SDKs.  I fixed this by accident, but it's possible customers had a dependency on the previous behavior.

Reply:
Definitely needs a changelog entry if we decide this is an acceptable behavioral break for a minor rev.

I'm leaning towards this being an acceptable change because even though this is the thing we tell people to automate their recovery based on, they wouldn't actually know what went wrong if they just got the HTTP status and not the string error code, so I'm not sure what there really is to silently break.

Reply:
Added a changelog comment.